src/widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -46,6 +54,10 @@ const renderer = ({
       shouldAutoHideContainer={shouldAutoHideContainer}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
+      showMore={showMore}
+      toggleShowMore={toggleShowMore}
+      isShowingMore={isShowingMore}
+      canToggleShowMore={true}
```

Comment:
The value should come from the connector right?

Reply:
Indeed it should be provided but the connector.